### PR TITLE
chore: update adding keys command

### PR DESCRIPTION
**Pull request by rabi-siddique**

### CODE REVIEW

File: test/e2e/support.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ Cypress.Commands.add('addKeys', params => {
   const { keyName, mnemonic, expectedAddress } = params;
   const command = `echo ${mnemonic} | agd keys add ${keyName} --recover ${keyRing}`;
 
-  cy.exec(command).then(({ stdout }) => {
+  cy.exec(command, {
+    failOnNonZeroExit: false,
+  }).then(({ stdout }) => {
```
Comment (frazarshad) [inline test/e2e/support.js:14]: can you update the PR description on why this change is needed?

Comment (frazarshad) [reply test/e2e/support.js:14]: also wont this flag also fail to throw an error if the command fails? resulting in the test falsely passing?

Comment (rabi-siddique) [reply test/e2e/support.js:14]: No. The test cases will fail if the command fails. The change is implemented to hide stack-traces.

**Review by frazarshad — APPROVED** [review]
